src/Stormpath/Stormpath.php

```diff
@@ -80,6 +80,9 @@ class Stormpath
     const ASCENDING                             = 'asc';
     const DESCENDING                            = 'desc';
 
+    const AUTHENTICATION_SCHEME_SAUTHC1         = 'SAuthc1';
+    const AUTHENTICATION_SCHEME_BASIC           = 'Basic';
```

Comment:
How about renaming these to put the scheme name at the beginning of the constant name (like BASIC_AUTHENTICATION_SCHEME)? I think it would be faster to read for the user.